### DISCUSSION TITLE
Больше професий

### DISCUSSION THREAD

**Discussion by travka4000** [octothorp-team/public-dobrograd#715] [Предложения]
**Думаю стоит на Доброград стоит добавить больше профессий, думаю это будет интересно. По типу мусорщика ведь наверно многие устали от привычных нам работ.**

**Comment by travka4000** [comment]
Предложение конечно очень короткое но думаю суть поняли

**Reply by VaR1aN** [reply]
Зачем больше профессий? Если хочешь убирать мусор, пойти в CSD например, они чинят город и убирают мусор.

**Comment by KOwOMei** [comment]
Зачем тебе куча профессий? Хочешь отыгрывать мусорщика - или в CSD. Они как раз и город чинят, и мусор убирают. Зачем вам нужны куча разных профессий, когда вы их можете обыгрывать через РП.

**Reply by travka4000** [reply]
ну простите, не знал, ведь я из-за бана много чего пропустил

**Comment by inquizzyoff** [comment]
В этом нет никакого смысла
